ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/maven/tasks/main.yml

```diff
@@ -16,7 +16,16 @@
    url: https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
    dest: /tmp/
    checksum: sha512:c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
-  when: maven_installed.rc != 0
+  when: maven_installed.rc != 0 and ansible_distribution != "Solaris"
+  tags: maven
+
+- name: Download Apache Maven v3.6.3 (Solaris)
+  get_url:
+   url: https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
```

Comment:
Same comment as freemarker ... why use a differnet method for these when using `wget` elsewhere?

Reply:
I've used get_url wherever possible. It seems to be down to the TLS version that each server accepts